tasks/server_keys.yml

```diff
@@ -66,10 +73,12 @@
     mode: 0400
 
 - name: Sign server key
-  command: openssl x509 -req -in server.csr -out server.crt -CA ca.crt -CAkey ca-key.pem -sha256 -days 3650 -CAcreateserial -extfile openssl-server.ext
+  command: "openssl x509 -req -in server.csr -out server.crt -CA ca.crt -CAkey ca-key.pem -sha256 -days 3650 -CAcreateserial -extfile openssl-server.ext -passin pass:$certkeypass"
```

Comment:
This looks like it's specific to your environment. The CA setup isn't password protected right now.